code/game/objects/items/devices/scanners.dm

```diff
@@ -364,6 +364,20 @@ FORENSIC SCANNER
 		holder = E.warhead.reagents
 		for(var/obj/container in E.warhead.containers)
 			scan(container)
+	else if(istype(O,/obj/structure/ship_ammo/rocket/custom_missile))
```

Comment:
```suggestion
	else if(istype(O, /obj/structure/ship_ammo/rocket/custom_missile))
```
spaces is good!!!!( I hope)

Reply:
ok but all others dont have spaces but sure